frontend/src/feature/cards/components/CardDetailsComponent.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <v-card
+    class="pa-2 ma-2 mx-auto d-flex flex-column justify-space-between" max-width="500px" min-height="300px">
+    <v-card-title>
+      {{ card.title }}
+    </v-card-title>
+    <v-card-subtitle>
+      <ul class="tags">
+        <li v-for="(tag, index) in card.tags">
+          <template v-if="index < card.tags!.length-1 ">
+            {{ tag }},
+          </template>
+          <template v-else>
+            {{ tag }}
+          </template>
+        </li>
+      </ul>
+    </v-card-subtitle>
+    <v-card-text>
+      <p>
+        {{ card.question }}
+      </p>
+      <ol>
+        <li v-for="(option, index) in card.options">
+          {{ index }}./{{ option }}
+        </li>
+      </ol>
+      <p>
+        Answer: {{ card.correctOption }}
+      </p>
+    </v-card-text>
+    <v-container>
+      <v-card-actions class="pa-0 ma-0">
+        <edit-mdi-button>Edit</edit-mdi-button>
+        <delete-mdi-button>Delete</delete-mdi-button>
+      </v-card-actions>
```

Comment:
no text in the buttons :-( we have tooltips - see code below. also move them into v-card-title